.travis.yml

```diff
@@ -2,56 +2,60 @@ language: cpp
 sudo: required
 dist: trusty
 
+notifications:
+  email: false
+
 matrix:
   include:
-    - env: ARCH=x86_64
-      addons:
-        apt:
-          update: true
-          packages:
-            - libmagic-dev
-            - libjpeg-dev
-            - libpng-dev
-            - cimg-dev
-            - automake  # required for patchelf
-            - gcc
-            - g++
-    - env: ARCH=i386
-      addons:
-        apt:
-          update: true
-          packages:
-            - libmagic-dev:i386
-            - libjpeg-dev:i386
-            - libpng-dev:i386
-            - automake  # required for patchelf
-            - libfuse2:i386
-            - libcairo2:i386
-            - gcc-multilib
-            - g++-multilib
+  - env: ARCH=x86_64
+    addons:
+      apt:
+        update: true
+        packages:
+        - libmagic-dev
+        - libjpeg-dev
+        - libpng-dev
+        - cimg-dev
+        - automake  # required for patchelf
+        - gcc
+        - g++
+  - env: ARCH=i386
+    addons:
+      apt:
+        update: true
+        packages:
+        - libmagic-dev:i386
+        - libjpeg-dev:i386
+        - libpng-dev:i386
+        - automake  # required for patchelf
+        - libfuse2:i386
+        - libcairo2:i386
+        - gcc-multilib
+        - g++-multilib
 
 install:
-  - git clone https://github.com/NixOS/patchelf.git -b 0.8
```

Comment:
Did you _have_ to change the indentation in yet _another_ project? The new version is way less readable IMO, and non-standard. I can't see what you really changed, but see

```diff
- [old blob]
+ [new blob]
```

Please switch to the old indentation so that a comparison is possible.

Reply:
just the indentation and the 
`
notifications:
  email: false
`
were changed

Reply:
Still, there is no reason to change the indentation to the worse. Please fix.